bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/i18n/souliss_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
+# FIXME: please do not add the file to the repo if you add or change no content
+# binding
```

Comment:
You can delete this file if you are not using it.

Reply:
removed